xml/storage_multipath.xml

```diff
@@ -646,9 +633,9 @@
        </entry>
        <entry>
         <para>
-         Maps linear devmaps to partitions on the multipathed device, which
-         makes it possible to create multipath monitoring for partitions on the
-         device.
+         Command line tool for managing &ldquo;partitions&rdquo; on multipath
```

Comment:
Because these devices aren't the same thing as SCSI disk partitions, for example. Those are first-class kernel devices, as the kernel has an internal concept of disks and partitions. The kernel "knows" that a certain partitions belongs to a certain disk. That concept doesn't apply to DM devices. kpartx "partitions" are just linear mappings on top of other DM devices. The kernel does not regard them as partitions of their parent device. They look like partitions to user space, and they map to exactly the same sectors as the partitions of the SCSI partitions of the underlying devices, but technically they are different.